core/app/[locale]/(default)/layout.tsx

```diff
@@ -16,15 +14,13 @@ export default async function DefaultLayout({ params, children }: Props) {
   setRequestLocale(locale);
 
   return (
-    <MakeswiftProvider previewMode={(await draftMode()).isEnabled}>
```

Comment:
Moved `<MakeswiftProvider>` to the root [root layout](https://github.com/bigcommerce/catalyst/pull/1791/files#diff-ed3d790da42bfec9d9674dffe6ce6cf4cace5dcdf3131247f416e8393a4836cfR99) so that theming can be applied to all pages.